Logout-Homepage View

ohylime opened the pull request:
Home page header and images rendered in component. So far this is what it looks like. Will be creating new branch after to work on footer so it doesn't cover the other components. 
![screenshot](https://cloud.githubusercontent.com/assets/19522302/24325889/13f9244c-1179-11e7-9140-93603b37a9ff.png)

ohylime committed:
basic html added

ohylime committed:
place holder image added into dist directory

ohylime committed:
Merge branch 'master' into logoutew

ohylime committed:
added imgs

ohylime committed:
Place holder images rendered on LoggedOutHomePage

covlllp commented on src/components/home/logged-out.jsx:
```diff
@@ -30,7 +29,25 @@ export default class LoggedOutHome extends React.PureComponent {
         <div className="body-with-title-bar container">
-          logged out wew
+        <div className="description">
+          <div className="header1">
```
make header1 and header2 a single h1 tag

ohylime replied:
changed~

covlllp commented on src/components/home/logged-out.jsx:
```diff
@@ -30,7 +29,25 @@ export default class LoggedOutHome extends React.PureComponent {
+          the alumni-powered
+          </div>
+          <div className="header2">scholarship</div>
+          <div> built by </div>
```
put this in a singular div, with two divs to break up two lines

ohylime replied:
okay

emilydev committed:
Indentation added 

Easier to read

covlllp commented on src/components/home/logged-out.jsx:
```diff
@@ -30,7 +29,25 @@ export default class LoggedOutHome extends React.PureComponent {
+          <div> fullstack academy alumni for future alumni
+          </div>
+        </div>
+          <div className="row">
```
for readability can you break this out to it's own submethod? Something like `renderImages`

Same with the header portion

ohylime replied:
submethods made!

covlllp commented on src/stylesheets/components/logged-out.less:
```diff
@@ -0,0 +1,19 @@
+  margin : 9rem
+}
+
+.header1 {
```
header1 and header2 don't need to be idfferent classes right?

ohylime replied:
combined them

ohylime committed:
Revisions made
Note : 1. Div tags changed to more specific tags
2. submethods made in logout view for readibility
3. unused classes removed

ohylime committed:
Merge branch 'logoutew' of https://github.com/fullstack-alumni/scholarship into logoutew

covlllp commented on src/components/home/logged-out.jsx:
```diff
@@ -25,12 +24,50 @@ export default class LoggedOutHome extends React.PureComponent {
+  renderLogoutViewHeader() {
+    const header = (
+      <div className="description">
+        <h1 className="header">
```
make this just one h1

covlllp commented on src/components/home/logged-out.jsx:
```diff
@@ -25,12 +24,50 @@ export default class LoggedOutHome extends React.PureComponent {
+  renderLogoutViewImages() {
+    const images = (
+      <div className="row">
+        <div className="col-sm-6 col-md-3 col-lg-3">
```
make this a function that you just call 3x

covlllp commented on src/components/home/logged-out.jsx:
```diff
@@ -25,12 +24,50 @@ export default class LoggedOutHome extends React.PureComponent {
         {this.renderTitleBar()}
         <div className="body-with-title-bar container">
-          logged out wew
+          {this.renderLogoutViewHeader()}
```
nit: don't include `Logout`, just have `renderHeader` and `renderImages`

covlllp commented on src/stylesheets/components/logged-out.less:
```diff
@@ -0,0 +1,14 @@
+.description {
+  text-align: center;
```
use the `.text-align` class that's made in `main.less`

covlllp reviewed (changes requested):
component cleanup

ohylime committed:
refactering made

emilydev committed:
Changes for code consistency

covlllp reviewed (approved):
lgtm
